Tests/QueueInteropTransportTest.php

```diff
@@ -83,8 +88,7 @@ public function testSendWithoutDebugWillNotVerifyTheInfrastructureForPerformance
     {
         $topic = 'topic';
         $queue = 'queue';
-        $message = new \stdClass();
-        $message->foo = 'bar';
+        $message = new Envelope('bar');
```

Comment:
The `Envelope` should wrap the message for transport: `$envelope = new Envelope($message);`
Then just it calling `->send($envelope)`.

Reply:
will try